.travis.yml

```diff
@@ -7,7 +7,14 @@ node_js:
   - 10.14.2
   - 12
   - 14
-  - node
+  - 15
+env:
+  - REACT_DIST=latest
+  - REACT_DIST=next
+  - REACT_DIST=experimental
+  # node15 compat
+  # Revisit once https://github.com/npm/cli/pull/2047 is released.
+  - npm_config_yes=true
```

Comment:
I think we can ditch this: https://github.com/npm/cli/issues/1935#issuecomment-717562445

Reply:
We might have to manually install that version of npm though (such as with `npm install --global npm@7`). Also the `env` section is duplicated, we only need one.